### PR TITLE
troopers16 presentation fixes

### PR DESCRIPTION
**Pull request by dawud**
- correction of some typos
- line breaks for readability
- remove trailing whitespace
- rebuild slides.pdf

### CODE REVIEW

File: presentations/troopers16/agenda.md [hunk 1]
```diff
@@ -28,15 +28,14 @@ BetterCrypto(.org) - Applied Crypto Hardening is born
   discussion, everything FOSS, public and auditable
 
 ## BetterCrypto (cont.)
-* Do at least something against the **Cryptocalypse**
+* Do at least something against the **Crypto-Apocalypse**
```
Comment (sebix) [inline presentations/troopers16/agenda.md]: *Cryptocalypse* is a widely used term, I'm sure this was intentionally.

**Comment by sebix** [comment]
Aside from the *Cryptocalypse*, it looks fine!

**Comment by azet** [comment]
Most of these are whitespace fixes? This has been already presented, recorded and the slides put online. I'm fine with fixing typos - though I question the necessity.

**Comment by dawud** [comment]
Well,I found the presentation useful, and it's quite probably going to be read by others (after being presented), so it seemed nice to fix a couple of typos. White space corrections were done automatically by vim, and I didn't put the spaces back.

**Pull request merged**